components/json/rs/src/lib.rs

```diff
@@ -329,3 +336,24 @@ pub fn convert_all_numbers_to_string(json: &str, path: &str) -> String {
         })
         .unwrap_or_else(|_| "".into())
 }
+
+pub fn sanitize_json_string(json: &str) -> ffi::JsonParseResult {
+    match serde_json::from_str::<serde_json::Value>(&json) {
+        Ok(v) => {
+            return ffi::JsonParseResult {
+                result: v.to_string(),
+                error: String::new(),
+                line: 0,
+                column: 0,
+            }
+        }
+        Err(err) => {
+            return ffi::JsonParseResult {
+                result: String::new(),
```

Comment:
is empty string a valid JSON?

Reply:
nope, it isn't: `EOF while parsing a value at line 1 column 0`
We preserve the same logic as in the other sanitizer implementations.